action/protocol/staking/liquidstake_indexer.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2023 IoTeX Foundation
+// This source code is provided 'as is' and no warranties are given as to title or non-infringement, merchantability
+// or fitness for purpose and, to the extent permitted by law, all liability for your use of the code is disclaimed.
+// This source code is governed by Apache License 2.0 that can be found in the LICENSE file.
+
+package staking
+
+import (
+	"math/big"
+)
+
+type (
+	// LiquidStakingIndexer defines the interface of liquid staking reader
+	LiquidStakingIndexer interface {
+		CandidateVotes(name string) *big.Int
```

Comment:
use `name` or `ownerAddr` as the query key? name of candidate may change over time

Reply:
yeah, `ownerAddr` should be right